### PR TITLE
Fix README links

### PR DESCRIPTION
**Pull request by saurabh-net**

### CODE REVIEW

File: stored_procedures/README.md [hunk 1]
```diff
@@ -11,6 +11,30 @@ you can reference it like the following:
 DECLARE next_ids ARRAY<INT64> DEFAULT [];
 CALL bqutil.procedure.GetNextIds(10, next_ids);
 ```
+## Using the stored procedures
+
+All stored procedures within this repository are available under the `bqutil` project on
+publicly shared datasets. Queries can then reference the shared procedures in the US multi-region via
+`bqutil.<dataset>.<procedure>()`.
```
Comment (danieldeleo) [inline stored_procedures/README.md:18]: We already know the dataset name is procedure so just add placeholder for the procedure name. 
E.g. bqutil.procedure.<procedure_name>()

File: stored_procedures/README.md [hunk 2]
```diff
@@ -11,6 +11,30 @@ you can reference it like the following:
...
+
+Procedures within this repository are also deployed publicly into every other region that [BigQuery supports](https://cloud.google.com/bigquery/docs/locations). 
+In order to use a procedure in your desired location outside of the US multi-region, you can reference it via a dataset with a regional suffix:
+
+`bqutil.<dataset>_<region>.<procedure>()`.
```
Comment (danieldeleo) [inline stored_procedures/README.md:23]: bqutil.procedure_<region>.<procedure_name>()

**Review by danieldeleo — CHANGES_REQUESTED** [review]

File: stored_procedures/README.md (hunk 2 above)
Comment (saurabh-net) [reply stored_procedures/README.md]: Done

File: stored_procedures/README.md (hunk 1 above)
Comment (saurabh-net) [reply stored_procedures/README.md]: Done

**Review by danieldeleo — APPROVED** [review]

**Pull request merged**